x-pack/test/functional/services/ml/settings_calendar.ts

```diff
@@ -18,17 +18,26 @@ export function MachineLearningSettingsCalendarProvider(
   const retry = getService('retry');
   const comboBox = getService('comboBox');
 
+  const localeCompare = (a: string, b: string) => a.localeCompare(b);
```

Comment:
I think this is not needed. The doc string for the `compareFn` of the `sort` function says
> Function used to determine the order of the elements. It is expected to return a negative value if the first argument is less than the second argument, zero if they're equal, and a positive value otherwise. If omitted, the elements are sorted in ascending, ASCII character order.

Since we're just comparing strings, the default sorting should be good enough.

Reply:
I hope [this](https://github.com/elastic/kibana/pull/178553/commits/9b11596c5876671dc8a9732cc5e58b2ced011a99) satisfies boss!